lib/schemaUtils.js

```diff
@@ -2414,12 +2414,15 @@ module.exports = {
         // Try and set fields for originalRequest (example.request)
         thisOriginalRequest.method = item.request.method;
         // setting URL
-        thisOriginalRequest.url = displayUrl;
+        const clonedItemURL = _.cloneDeep(item.request.url);
+
+        thisOriginalRequest.url = clonedItemURL;
+        thisOriginalRequest.url.variable = clonedItemURL.variables.members;
```

Comment:
@Akshay2402 Can we mention here in a comment why we are overriding SDK defined `url.variable` object?

Reply:
Sure.